cvpy/tests/test_image.py

```diff
@@ -175,7 +176,79 @@ def test_get_image_array_const_ctype(self):
         image_array = Image.get_image_array_const_ctype(medical_binaries, medical_dimensions, medical_resolutions, ctype='8U', n=0, channel_count=1)
         
         self.assertTrue(np.array_equal(image_array, np.array([[0, 0, 0, 0, 0],[0, 255, 0, 0, 0],[0, 255, 0, 150, 0],[0, 0, 0, 0, 50],[0, 0, 0, 0, 0]])))
-        
+
+    def test_convert_wide_to_numpy_3_channel(self):
+        self.s = swat.CAS(self.casHost, self.casPort, self.username, self.password)
+        self.s.loadactionset('image')
+
+        # Add caslib
+        self.s.addcaslib(name='dlib', activeOnAdd=False, path=self.dataPath, dataSource='PATH', subdirectories=True)
+
+        # Load the image
+        image = self.s.CASTable('image', replace=True)
+        self.s.image.loadImages(path='images/Sas_c.jpg',
+                                casOut=dict(name='image', replace='TRUE'),
+                                addColumns={"WIDTH", "HEIGHT"},
+                                caslib='dlib',
+                                decode=True)
```

Comment:
Since WIDE encoding is supposed to be used in ESP, we should add ESP unit and integration test for the new Image.convert_wide_to_numpy function. 

We should keep these two tests in CVPy as well.

Reply:
I'm planning on creating a follow-up story to refactor some the ESP unit tests to use CVPy (and this function specifically). I haven't looked at the ESP tests yet but I'll add definitely add new tests if this functionality won't be covered.